### ISSUE TITLE
WARNING in webpack-subresource-integrity

### ISSUE THREAD

**Issue by vlarop** [waysact/webpack-subresource-integrity#129]
Cannot determine asset for chunk scripts, computed="undefined", available=scripts.859e64cef1ef944f8beb.js. Please report this full error message along with your Webpack configuration at https://github.com/waysact/webpack-subresource-integrity/issues/new

**Comment by jscheid** [comment]
Thanks, this happens with older angular-cli versions which depend on an old version of this plugin. Upgrading to angular-cli 10 should fix it.

Closing as a duplicate of #104

**Comment by vlarop** [comment]
Hi,
thanks for your answer, but I started using Angular 10 and suddenly I am
getting warnings:
WARNING in
C:\Users\Vladimira\pronajem-jizni-morava\node_modules\firebase\firestore\dist\index.esm.js
depends on '@firebase/firestore'. CommonJS or AMD dependencies can cause
optimization bailouts.
For more info see:
https://angular.io/guide/build#configuring-commonjs-dependencies
I did not see such warnings with older versions of Angular.
Best regards
Vladimira Loksova





po 12. 10. 2020 v 11:02 odesílatel Julian Scheid <notifications@github.com>
napsal:
